CRM/CiviAwards/Helper/CreateReviewFieldsValidator.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * Class CRM_CiviAwards_Helper_CreateReviewFieldsValidator.
+ */
+class CRM_CiviAwards_Helper_CreateReviewFieldsValidator {
+
+  /**
+   * Array of accepted review_fields with their validation filters
+   *
+   * @var array
+   */
+  private $reviewFieldsLegalOptions = [
+    'id' => FILTER_VALIDATE_INT,
+    'weight' => FILTER_VALIDATE_INT,
+    'required' => FILTER_VALIDATE_BOOLEAN
+  ];
+
+  /**
+   * Array to hold invalid field options
+   *
+   * @var array
+   */
+  private $invalidReviewFieldOptions = [];
+
+  /**
+   * Validates review_field options to ensure legal options are passed.
+   *
+   * @param array $reviewFields
+   *  Review fields
+   *
+   * @throws \Exception
+   */
+  public function validate(array $reviewFields) {
+    if (empty($reviewFields)) {
+      return;
+    }
+
+    foreach ($reviewFields as $reviewFieldOptions) {
+      $validReviewFieldOptions = array_filter($reviewFieldOptions, [
+        $this,
```

Comment:
Can you please make sure what is passed as the second parameter is on a single line
```php
[$this, 'reviewFieldOptionsFilter']
```
Thanks.